tests/accordion-test.ts

```diff
@@ -0,0 +1,414 @@
+                        <span>title 5</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 5</div>
+                </div>
+            </aalam-accordion>
+        `);
+
+        const activeelements = el.querySelectorAll(".acc-title");
+        console.log(activeelements.length);
+
+        expect(el.querySelectorAll(".acc-active").length).equal(0);
+        const element = el.querySelectorAll(".acc-title");
+        element[0]?.click();
+        element[1]?.click();
+        expect(el.querySelectorAll(".acc-active").length).equal(2);
+        element[0]?.click();
+        element[1]?.click();
+        expect(el.querySelectorAll(".acc-active").length).equal(1);
+
+        const ele = await fixture(html`
+            <aalam-accordion nocloseall activecls="acc-active">
+                <div class="container acc-active">
+                    <div class="acc-title">
+                        <span>What is Artificial Intelligence?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">
+                        <p>body 1</p>
+                    </div>
+                </div>
+                <div class="container acc-active">
+                    <div class="acc-title">
+                        <span>What is Machine Learning?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 2</div>
+                </div>
+                <div class="container">
+                    <div class="acc-title">
+                        <span>title 3?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 3</div>
+                </div>
+                <div class="container">
+                    <div class="acc-title">
+                        <span>title 4</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 4</div>
+                </div>
+                <div class="container">
+                    <div class="acc-title">
+                        <span>title 5</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 5</div>
+                </div>
+            </aalam-accordion>
+        `);
+        const activeitem = ele.querySelectorAll(".acc-title");
+        expect(ele.querySelectorAll(".acc-active").length).equal(2);
+
+        activeitem[0]?.click();
+        activeitem[1]?.click();
+
+        expect(ele.querySelectorAll(".acc-active").length).equal(1);
+
+        activeitem[0]?.click();
+        activeitem[1]?.click();
+        activeitem[2]?.click();
+        activeitem[1]?.click();
+        activeitem[3]?.click();
+        expect(ele.querySelectorAll(".acc-active").length).equal(4);
```

Comment:
You can take reference from the comment in the nomulitple case. Better to check with the display and the class names, here. Also set and remove the attribute on the same element and validate their behaviour in each case.